q2_diversity_lib/tests/test_beta.py

```diff
@@ -331,6 +332,43 @@ def test_accepted_types_have_consistent_behavior(self):
                                             self.expected[id1, id2])
 
 
+class BetaPhylogeneticMetaPassthroughTests(TestPluginBase):
+    package = 'q2_diversity_lib.tests'
+
+    def setUp(self):
+        super().setUp()
+        self.method = self.plugin.actions['beta_phylogenetic_meta_passthrough']
+        empty_table = biom.Table(np.array([]), [], [])
+        self.empty_table = Artifact.import_data('FeatureTable[Frequency]',
+                                                empty_table)
+
+        # checking parity with the unifrac.meta tests
+        table1 = pkg_resources.resource_filename('unifrac.tests',
+                                                 'data/e1.biom')
```

Comment:
Do you think we should copy these test data over to this plugin?

Reply:
No preference. This *might* be the regular crawford data actually